mvrx/src/main/kotlin/com/airbnb/mvrx/RealMvRxStateStore.kt

```diff
@@ -130,11 +128,11 @@ class RealMvRxStateStore<S : Any>(initialState: S) : MvRxStateStore<S> {
      * Coalesce all updates on the setState queue and clear the queue.
```

Comment:
comment `Coalesce all updates ` doesn't apply anymore?

Might be worth having a comment documenting why we chose to not coalesce